src/layouts/SiteLaunchPad/SiteLaunchPad.tsx

```diff
@@ -160,6 +161,29 @@ export const SiteLaunchPad = (): JSX.Element => {
     }
   }
 
+  const handleIncrementStepNumber = () => {
+    if (
+      siteLaunchStatusProps &&
+      siteLaunchStatusProps.stepNumber < SITE_LAUNCH_TASKS_LENGTH
+    ) {
+      setSiteLaunchStatusProps({
+        ...siteLaunchStatusProps,
+        stepNumber: (siteLaunchStatusProps.stepNumber +
+          1) as SiteLaunchTaskTypeIndex, // safe to assert since we do a check
+        siteLaunchStatus: "CHECKLIST_TASKS_PENDING",
+      })
+    }
+  }
+  const handleDecrementStepNumber = () => {
+    if (siteLaunchStatusProps && siteLaunchStatusProps.stepNumber > 0) {
+      setSiteLaunchStatusProps({
+        ...siteLaunchStatusProps,
+        stepNumber: (siteLaunchStatusProps?.stepNumber -
+          1) as SiteLaunchTaskTypeIndex, // safe to assert since we do a check
+      })
+    }
+  }
```

Comment:
These are duplicate definitions? Will need to remove one of them

Reply:
arh sorry, this was due to rebasing, somehow it duplicated the functions, fixed here 8133a6842261bb2da7ed7c31617fb7bba0e73f01